tests/performance_tests/test_forward_model_script_overhead.py

```diff
@@ -11,7 +11,7 @@
 )
 
 
-@pytest.mark.timeout(30)
+@pytest.mark.timeout(120)
```

Comment:
This is quite a difference. Could you comment on why make it 4x longer?

Reply:
changed to 60 seconds!